framework/base/Component.php

```diff
@@ -13,7 +13,7 @@
  * Component is the base class that implements the *property*, *event* and *behavior* features.
  *
  * Component provides the *event* and *behavior* features, in addition to the *property* feature which is implemented in
- * its parent class [[Object]].
+ * its parent class [[\yii\base\Object|Object]].
```

Comment:
is `Object` not resolved in `yii\base` namespace? if not I'd consider that a bug in apidoc generator.